modules/ai-operators.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * ai_workloads/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ai-operators_{context}"]
+= Operators for running AI workloads
+
+You can use Operators to run artificial intelligence (AI) and machine learning (ML) workloads on {product-title}. With Operators, you can build a customized environment that meets your specific AI/ML requirements while continuing to use {product-title} as the core platform for your applications.
+
+{product-title} provides several Operators that can help you run AI workloads:
+
+{lws-operator}::
+You can use the {lws-operator} to enable large-scale AI training workloads to run reliably across nodes with synchronization between leader and worker processes. Without proper coordination, large training runs might fail or stall.
```

Comment:
I think you want to focus more on inferencing workloads rather than training. LWS isn't really geared towards distributed training. It looks like the rest of your doc focuses on that so this just may be a typo.

Reply:
That was pulled from a write-up that PM gave me, but I'll adjust it, thanks!